### PR TITLE
Add Azure Stack HCI SDK

### PR DESCRIPTION
**Pull request by mbialecka**
1. Added generated management SDK for Microsoft.AzureStackHCI
2. Added a basic test.

### CODE REVIEW

File: sdk/azurestackhci/Microsoft.Azure.Management.AzureStackHCI/src/Microsoft.Azure.Management.AzureStackHCI.csproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <Description>Microsoft Azure Stack HCI Management Library</Description>
+    <Version>1.0.0</Version>
```
Comment (isra-fel) [inline sdk/azurestackhci/Microsoft.Azure.Management.AzureStackHCI/src/Microsoft.Azure.Management.AzureStackHCI.csproj:8]: We usually use `1.0.0-preview.1` as the version of the first release. Then `1.0.0-preview.2`, `1.0.0-preview.3` ...
When we are sure it's stable, change it to `1.0.0`

Read more about versioning guide: https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md

**Review by isra-fel — COMMENTED** [review]
Generally LGTM. Just one suggestion about the package version.

File: sdk/azurestackhci/Microsoft.Azure.Management.AzureStackHCI/src/Microsoft.Azure.Management.AzureStackHCI.csproj (hunk 1 above)
Comment (mbialecka) [reply sdk/azurestackhci/Microsoft.Azure.Management.AzureStackHCI/src/Microsoft.Azure.Management.AzureStackHCI.csproj]: Fixed, thanks!

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**